packages/firebase_admob/lib/firebase_admob.dart

```diff
@@ -82,6 +82,65 @@ class MobileAdTargetingInfo {
   }
 }
 
+// The types of ad sizes supported for banners.
+enum AdSizeType {
+  WidthAndHeight,
+  SmartBanner,
+}
+
+/// [AdSize] represents the size of a banner ad.
```

Comment:
It would be useful to link to the relevant doc here; https://developers.google.com/admob/android/banner#banner_sizes I think

Reply:
Done.